### DISCUSSION TITLE
French language ?

### DISCUSSION THREAD

**Discussion by billmetangmo** [chaskiq/chaskiq#830] [Q&A] [answered]
Hello, 

I just created an account on https://chaskiq.io/ but i don't see french as language as an option.
But from this blog article this seems to be supported: https://blog.chaskiq.io/chaskiq-0-8-5/

**Comment by michelson** [comment, marked as answer]
Hi @billmetangmo, you are absolutely right, French was not added, we will deploy that right away. 
Anyway it still needs some strings to be translated as is completed at 87%, if you feel that you could help us with the FR translations here is where we do that https://chaskiq.crowdin.com/chaskiq , its an easy service to update the missing translations

**Reply by billmetangmo** [reply]
I translated the remaining 13% @michelson
